harmonizer-2/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/apollographql/federation/"
 license-file = "./LICENSE"
 readme = "README.md"
 include = [ "dist/**/*", "deno/**/*", "src/**/*", "snapshots/**/*", "Cargo.toml" ]
+exclude = [ "snapshots/query_runtime.snap" ]
```

Comment:
should likely exclude `snapshots/**/*` from the `include` here, i added it while you weren't looking bc i didn't realize they were architecture dependent